ssh-key/src/signature.rs

```diff
@@ -7,7 +7,10 @@ use encoding::{CheckedSum, Decode, Encode, Reader, Writer};
 use signature::{Signer, Verifier};
 
 #[cfg(feature = "ed25519")]
-use crate::{private::Ed25519Keypair, public::Ed25519PublicKey};
+use {
+    crate::{private::Ed25519Keypair, public::Ed25519PublicKey},
+    sha2::{Digest, Sha256},
```

Comment:
You need to consolidate these imports with the ones for the `rsa` crate.

Reply:
I noticed the lints but haven't had the time to fix them. I have pushed a new commit now.

All checks seem to pass now.